README.md

```diff
@@ -18,8 +18,8 @@ The Kanban-like-task-management project has two interfaces - the Board listing,
 
 ### Pre-requisites
 
-1. Aras Innovator installed (version 11.0 SPx preferred)
-2. Aras Package Import tool
+1. Aras Innovator installed (version 11.0 SPx preferred).

```

Comment:
We don't really need a period at the end of this line (or the next) because it's just a list of tool/installation requirements, not a list of sentences.

Reply:
@EliJDonahue I somehow missed your comment until now. Fixed!